### PR TITLE
Added Activities Vs NationalParks Donut Chart 

### PR DESCRIPTION
**Pull request by juhiaugustine**
This PR adds a Donut Chart that displays the comparison between different activities and the number of associated national parks. 

**Future work:**
I am planning to create a pop-up modal view for each activity. This modal view will display a list of park links that would redirect to the parkInfoPage. This enhancement will provide users with more detailed information about each activity and the corresponding national parks.

image

### CODE REVIEW

File: park-dashboard/src/Pages/NationalParkCharts/ActivitiesVsNP.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  let data = {
+    labels: Object.keys(activitiesObject),
+    datasets: [
+      {
+        label: "Number of National Parks",
+        data: Object.values(activitiesObject).map((parks) => {
+          return parks.length;
+        }),
+        backgroundColor: [
```
Comment (rutujapadgilwar) [inline park-dashboard/src/Pages/NationalParkCharts/ActivitiesVsNP.jsx]: Add more colors so colors can't be repeated.

**Review by rutujapadgilwar — CHANGES_REQUESTED** [review]
Hello Juhi, the donut chart looks fine, but I feel we need to add unique colors for each activity. Moreover, add the ActivitiesVsNp.jsx file to the NationalParkActivities component. 
other things look good to me.

File: park-dashboard/src/Pages/NationalParkCharts/ActivitiesVsNP.jsx (hunk 1 above)
Comment (juhiaugustine) [reply park-dashboard/src/Pages/NationalParkCharts/ActivitiesVsNP.jsx]: I have hard coded unique colors for each activity.

image

File: park-dashboard/src/Components/NationalParkActivities/ActivitiesVsNP.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      .then((res) => {
+        const activities = res.data.data;
+
+        let activitiesObject = {};
+
+        activities.forEach((activity) => {
+          const parks = activity.parks.filter(
+            (park) => park.designation === "National Park"
+          );
+          if (parks.length < 15) return;
```
Comment (schoure98) [inline park-dashboard/src/Components/NationalParkActivities/ActivitiesVsNP.jsx:36]: Good work with selecting the popular activities by setting the limit.

**Review by schoure98 — COMMENTED** [review]
Can you keep the color intensity same as the before? I think the new colors are looking a bit light.

**Comment by juhiaugustine** [comment]
> Can you keep the color intensity same as the before? I think the new colors are looking a bit light.

I have increased the intensity of the colors now. Let me know how it looks now?

image

**Pull request merged**